src/widgets/search.widget.tsx

```diff
@@ -152,14 +143,23 @@ export const SearchWidget: FC<Props> = observer(({style}) => {
               !store.ui.query && section.key !== 'favorites'
                 ? store.ui.favoriteItems.length + index
                 : index
+
+            if (item.type === ItemType.TEMPORARY_RESULT) {
```

Comment:
This takes care of the UI part but probably not of the enter key handler, you also need to adjust the behavior there (ui.store -> onKeyDown)